impls/src/lifecycle/seed.rs

```diff
@@ -203,7 +202,7 @@ impl WalletSeed {
 				 Run \"grin-wallet init\" to initialize a new wallet.",
 				seed_file_path
 			);
-			Err(ErrorKind::WalletSeedDoesntExist)?
+			Err(ErrorKind::WalletSeedDoesntExist.into())
```

Comment:
Why does this thing hate the ? operator so much?

Reply:
^^ it’s just more straightforward to return directly the error.